### ISSUE TITLE
Creating snap package

### ISSUE THREAD

**Issue by nzt** [roswell/roswell#425]
I created a snap recipe for roswell. I believe this recipe makes installation to be easily.
As some CI service supports deployment of this kind of packages, we can upload latest package synchronously with GitHub Releases.

We have two options;
- Add CI script to deploy package
- Create new repository to manage new package.

Second option does not need to be a official repository, that is, I might create unofficial package if you want to make no more new format of package.

> Snap is a software deployment and package management system developed by Canonical for the Linux operating system. The packages, called snaps, and the tool for using them, snapd, work across a range of Linux distributions allowing distribution-agnostic upstream software packaging. Snap was originally designed for Ubuntu Touch. The system is designed to work for internet of things, cloud and desktop computing. (Wikiepdia, https://en.wikipedia.org/wiki/Snap_(package_manager))

**Comment by snmsts** [comment]
You may...I believe I give enough authority to do whatever. I think it might be better to register to snapcraft.io  directly than have file here.

**Comment by nzt** [comment]
Yes, your right. You have already given me authority to make repository. However, I'd like to get your agreement to make new repository in this organization. 
I'll create a new repository to host snap package recipe and to fire CI event to upload snapcraft.io

**Comment by snmsts** [comment]
I agreed.
